### PR TITLE
V10: Healthcheck documentation. 

### PR DESCRIPTION
**Pull request by bergmania**
Updated healthcheck documentation. Some code examples and links are updated to v10, but overall there should not be any differences between v9 and v10, so I marked all documents to be valid for both versions

### CODE REVIEW

File: Extending/Health-Check/index.md [hunk 1]
```diff
@@ -24,6 +25,7 @@ Umbraco comes with the following checks by default:
 * Category **Permissions**
   * **Folder & File Permissions (id: `53DBA282-4A79-4B67-B958-B29EC40FCC23`)** - checks that folders and files that are either required or recommended to set with write permissions can be accessed
```
Comment (eshanrnh) [inline Extending/Health-Check/index.md:26]:
```suggestion
  * **Folder & File Permissions (id: `53DBA282-4A79-4B67-B958-B29EC40FCC23`)** - checks that the folders and files that are either required or recommended set with write permissions can be accessed
```

File: Extending/Health-Check/index.md [hunk 2]
```diff
@@ -57,54 +59,68 @@ Each health check is a class that needs to have a `HealthCheck` attribute. This
 
 ### Configuration checks
 
-These are small checks that take an XPath query and confirm that the value that's expected is there. If the value is not correct, clicking the "Rectify" button will set the recommended value.
+These are small checks that take an [IConfiguration](https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.configuration.iconfiguration?view=dotnet-plat-ext-6.0) key and confirm that the value that's expected is there. If the value is not correct, there will link to a guide for how to set this value correct.
 
 * A configuration check needs to inherit from `Umbraco.Cms.Core.HealthChecks.Checks.AbstractSettingsCheck`
 * A configuration check needs the `HealthCheck` attribute as noted at the start of this document
 * `ReadMoreLink` is a link to an external guide which will help you to troubleshoot any problems
```
Comment (eshanrnh) [inline Extending/Health-Check/index.md:66]:
```suggestion
* `ReadMoreLink` is a link to an external guide that will help you to troubleshoot any problems
```

File: Extending/Health-Check/index.md [hunk 3]
```diff
@@ -334,36 +351,41 @@ namespace Umbraco.Cms.Core.HealthChecks.NotificationMethods
...
-            var isBodyHtml = message.IsNullOrWhiteSpace() == false && message.Contains("<") && message.Contains("</");
+            var isBodyHtml = message.IsNullOrWhiteSpace() == false && message!.Contains("<") && message.Contains("</");
             return new EmailMessage(to, RecipientEmail, subject, message, isBodyHtml);
         }
     }
 }
+
 ```
 
 If a custom configuration is required for a custom notification method, the following extract can be merged in `appsettings.json` file, which will enable the email notification method to be configured:
```
Comment (eshanrnh) [inline Extending/Health-Check/index.md:391]:
```suggestion
If a custom configuration is required for a custom notification method, the following extract can be merged in the `appsettings.json` file, which will enable the email notification method to be configured:
```

**Comment by bergmania** [comment]
@eshanrnh, feel free to just update everything grammar wise on all my docs PRs ;) I have no opinions about grammar. Not even in  my native language 🙈

**Comment by eshanrnh** [comment]
Haha, sounds good. I'll take care of it 💪

**Pull request merged**